pages/services/storage/terminology-and-concepts/index.md

```diff
@@ -37,29 +37,31 @@ A gRPC endpoint that implements the CSI Services specified in the CSI spec. See
 
 ## Storage Local Resource Provider (SLRP)
 
-A Mesos component that drives a single CSI plugin that manages storage resources that exist on a single agent. It presents these storage resources (both raw capacity and in the form of volumes) to Mesos and translates operations on those resources to CSI RPCs which it executes against the CSI plugin. Examples of such CSI plugins are those for LVM2, Raw GPT, other direct-attached storage. Learn more details about SLRP in this <a href ="http://mesos.apache.org/documentation/latest/csi/#storage-local-resource-provider/">Mesos documentation</a>.
+A Mesos component that drives a single CSI plugin which manages storage resources that exist on a single agent. It presents these storage resources (both raw capacity and in the form of volumes) to Mesos and translates operations on those resources to CSI RPCs which it executes against the CSI plugin. Examples of CSI plugins are those for LVM2, Raw GPT, other direct-attached storage. Learn more details about SLRP in <a href ="http://mesos.apache.org/documentation/latest/csi/#storage-local-resource-provider/">Mesos documentation</a>.
```

Comment:
is anyone actually providing a GPT plugin at this point? if so, I wonder how they deal with the fragmentation problem